src/main/java/com/buddy/tutorial/model/User.java

```diff
@@ -0,0 +1,17 @@
+package com.buddy.tutorial.model;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+import lombok.Data;
+
+@Entity(name = "Users")
```

Comment:
verify table and fields follows naming  conventions. refer https://www.sqlshack.com/learn-sql-naming-conventions/      https://www.geeksforgeeks.org/database-table-and-column-naming-conventions/